bioconda_utils/lint_functions.py

```diff
@@ -333,7 +357,5 @@ def pin(recipe, meta, df):
     _pin('CONDA_HDF5', 'hdf5'),
     _pin('CONDA_NCURSES', 'ncurses'),
     _pin('CONDA_HTSLIB', 'htslib'),
-
-    # Pinning in env_matrix.yml is not consistent for bzip2.
-    # _pin('CONDA_BZIP2', 'bzip2'),
+    _pin('CONDA_BZIP2', 'bzip2'),
```

Comment:
I think that need to be special-cased in the code above. Maybe also a specific test should be added.

Reply:
Why? The wrong entry in the env matrix was fixed in PR bioconda/bioconda-recipes#6310

Reply:
Ah, sorry, I missed that PR!